app/ceom/aboutus/views.py

```diff
@@ -18,9 +18,9 @@ def people(request):
     
     return render(request, 'aboutus/people.html', context=data)
 
-def index(request):
+def publications(request):
+    print("Hi:)")
```

Comment:
Never put print statements like this into production.
It's fine to use them for debugging, you can print whatever you want for debugging, but I will rarely ever allow a print statement to go into production unless absolutely necessary. (In the entire website there are only 1 or 2 print statements that I have allowed)

The reason is it pollutes the logs for everyone else. If everyone did this, the logs would constantly print out garbage from a hundred different views and we would have a hell of a time finding error messages or the print statements we're actually looking for.

(Also for clarification on tone, I'm not upset, it's an easy thing to miss and this is otherwise good work.)